src/subscription_manager/gui/data/glade/installed.glade

```diff
@@ -3,6 +3,19 @@
   <!-- interface-requires gtk+ 2.10 -->
   <!--
  interface-naming-policy project-wide -->
+  <object class="GtkAdjustment" id="adjustment1">
+    <property name="upper">396</property>
+    <property name="step_increment">39.6</property>
+    <property name="page_increment">356.4</property>
+    <property name="page_size">396</property>
+  </object>
+  <object class="GtkAdjustment" id="adjustment2">
```

Comment:
adjustment1 et al is the default glade naming scheme, maybe something like 'scrolled_window_3_h_adjustment' and 'scrolled_window_3_v_adjustment'?  (ala, gui/data/ui/installed.ui)